style.css

```diff
@@ -383,3 +393,124 @@ header {
   background: #6070ff;
   color: #fff;
 }
+
+.nav-desktop {
+  display: none;
+}
+
+@media all and (min-width: 768px) {
+  /* ===============================*\
+           Header
+*\=============================== */
+
+  .nav-desktop {
+    display: block;
+  }
+
+  .menu-desktop {
+    list-style: none;
+    display: flex;
+    gap: 20px;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  header .navbar .menu-desktop a {
+    font-size: 15px;
+    line-height: 20px;
+    font-style: normal;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  header .navbar .menu-desktop a:hover {
+    color: #6070ff;
+  }
+
+  .hamb-bu {
+    display: none;
+  }
+
+  .navbar {
+    padding: 15px 142px;
+  }
+
+  /* ===============================*\
+           headline
+*\=============================== */
+
+  #headline {
+    background-image: url("images/Header-Desktop.svg");
+    justify-content: center;
+    padding: 0 110px;
+  }
+
+  #headline h1 {
+    font-size: 48px;
+  }
+
+  #headline p {
+    font-size: 20px;
+  }
+
+  /* ===============================*\
+           Work
+*\=============================== */
+  .portfolio-snapshot {
+    flex-direction: row;
+  }
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. Please increase the margin between the border of your card and the title. :eyes: :eyes: 
Your design
![image](https://user-images.githubusercontent.com/98363075/200099035-a2bb792b-fa39-4574-8b71-410c5f381a7d.png)

Figma Template
![image](https://user-images.githubusercontent.com/98363075/200099050-0ca70b53-b930-4f5c-88b6-90c6662ea3cb.png)